roles/rsyslog/roles/input_roles/files/tasks/cleanup.yaml

```diff
@@ -1,10 +1,8 @@
 ---
-- name: Set input files facts
-  set_fact:
-    rsyslog_role_packages: "{{ rsyslog_files_input_packages|flatten }}"
-    rsyslog_role_rules: "{{ rsyslog_files_input_rules }}"
-
+# Deploy configuration files
 - name: Remove input files example config files from rsyslog.d
+  vars:
+    __rsyslog_rules: "{{ rsyslog_files_input_rules }}"
```

Comment:
@nhosoi You don't need __rsyslog_packages here?

Reply:
Well, we don't have a package cleanup code in cleanup.yaml. [1]  So, nothing happens even if we pass __rsyslog_packages...  Do you think we should uninstall packages?  If we uninstall rsyslog, it cleans up everything including the existing config files.  Is it preferable?  I checked linux-system-roles/{network,timesync,selinux}.  Network does not call "package"; the other 2 install packages with present, but I don't see absent...

[1] - https://github.com/linux-system-roles/logging/blob/master/roles/rsyslog/tasks/cleanup.yaml

Reply:
@nhosoi No. You are right. I missed that this is the cleanup.yaml

Reply:
FYI I don't think that we need an option to unistall the packages that we have installed. @nhosoi  As you observed, other roles don't do it either.

Reply:
Thanks for your input, @pcahyna.